### ISSUE TITLE
about multi-gpu

### ISSUE THREAD

**Issue by Kotori05** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#364]
Advice Request 
tiled vae is really a good idea, is it possible to use on multiple GPU environment? (cuda out of memory!)

**Comment by Kahsolt** [comment]
Yes multi-GPU can be helpful for tiled-VAE, since the main bottleneck for tiled-VAE forward is the GroupNorm sync :)
but the sd-webui seems not to handle multi-GPU case, it is not considered to impl that in this repo
